index.html

```diff
@@ -1,277 +1,268 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+
+  <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Alex Portfolio</title>
-</head>
-<body>
+  </head>
+
+  <body>
     <!--Principal nav-->
     <header>
-        <nav>
-          <a href="">Alexander</a>
-          <ul>
-            <li class="nav-list menu-collapsed" ><a href="">
-                <div class="menu-line"></div>
-                <div class="menu-line"></div>
-                <div class="menu-line"></div>
-            </a></li>
-          </ul>
-          
-          <ul class="navbar-items">
-            <li class="nav-list"><a href="">Portfolio</a></li>
-            <li class="nav-list"><a href="">About</a></li>
-            <li class="nav-list"><a href="">Contact</a></li>
-           
-          </ul>
-        </nav>
+      <nav class="nav-style">
+        <a href="#" class="text-logo" id="logo">Alexander</a>
+
+        <button type="button" id="menu-container" onclick="togglerMenu()">
+          <span class="open-menu menu-collapsed" id="menu-button"></span>
+        </button>
```

Comment:
- [ ] I can't see your close button please try to fix that  so when the menu is open  the close button should be displayed

Reply:
My close is visible. I don't know why you don't see it.
it on line 574 to +579

Reply:
> My close is visible. I don't know why you don't see it. it on line 574 to +579

@masangana  but it wasn't appearing on the website because you don't use `./` in the  URL of the image maybe it will display on your machine but not on everyone's
I suggest using the dot and slash `./` that indicates to look in the same directory